core-api/src/main/java/com/optimizely/ab/bucketing/DecisionService.java

```diff
@@ -343,4 +337,25 @@ void saveVariation(@Nonnull Experiment experiment,
             }
         }
     }
+
+    /**
+     * @param userId The userId of the user.
```

Comment:
Please provide a description for this method. Also, please make it private as it is not used outside this class.